### ISSUE TITLE
Test plans don't work again with the same error

### ISSUE THREAD

**Issue by vdavydovHH** [lyndsey-ferguson/fastlane-plugin-test_center#243]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

> If you love this fastlane plugin, consider sponsoring it or asking your company to sponsor it. I would really appreciate any
> gesture: https://github.com/sponsors/lyndsey-ferguson. 😍

### Issue Description
Hi @lyndsey-ferguson, it's me again)

In 3.11.4 release test plans have broken again with the same error `xcodebuild: error: The flag -testPlan <name> cannot be used since the scheme does not use test plans.` Could you see what happened? I've attached sample app, but i think any of  the earlier samples can help you. If you need something else,  let me know please ;)

Thanks in advance!

[MultiScanTestPlans 2.zip](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/4846642/MultiScanTestPlans.2.zip)

**Comment by lyndsey-ferguson** [comment]
Thanks @vdavydovHH , this is so embarrassing. Hopefully, I can fix it tonight.

**Comment by lyndsey-ferguson** [comment]
The [PR](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/236) for fixing how sim logs are collected caused this problem.

**Comment by lyndsey-ferguson** [comment]
@vdavydovHH can you test this? It was a foolish mistake and I fixed it quickly.

Modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-243-testplans-not-found-again"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Actually, I need to fix this right away, and I know that this code needs to be changed. So test it to confirm for me, I have to merge it now.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.11.5](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.11.5)
